insights/parsers/logrotate_conf.py

```diff
@@ -116,7 +116,7 @@ def _parse_opts(line):
                     self.options.append(key1)
             else:
                 # in log_file section
-                if line.endswith('}'):
+                if line.endswith('}') and script is None:
```

Comment:
I can also see that there is a `"} >>${ACTLOG_RLOG} 2>&1` in your sample. I'm afraid that it will cause that the parsed lines cannot be saved correctly. Please have a check.

Reply:
@xiangce, it is good. I added the data in test too. Let me check this line too in the test.